### PR TITLE
Address final UX issues

### PR DESCRIPTION
**Pull request by andy-armstrong**
## [TNL-3936](https://openedx.atlassian.net/browse/TNL-3936)

Address the remaining UX issues found by Frances and Christina (see the review here: https://github.com/edx/edx-ora2/pull/824)

### Sandbox
- [x] https://andy-armstrong.sandbox.edx.org/

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens 
- [x] Code review: @dianakhuang 

### Post-review
- [ ] Squash commits

### CODE REVIEW

File: openassessment/templates/openassessmentblock/response/oa_response.html [hunk 1]
```diff
@@ -109,22 +109,22 @@ <h3 class="response__submission__status__title">
...
+                                        <h3 class="response__submission__status__title">
+                                            {% trans "Click to preview your submission in LaTeX"%}
+                                        </h3>
                                     </div>
-                                </article>
-                            </li>
+                                </li>
+                                <li id = "submission__preview__item" class="list--actions__item">
+                                    <article class="submission__answer__display">
+                                        <h3 class="submission__answer__display__title">Preview Response</h3>
```
Comment (dianakhuang) [inline openassessment/templates/openassessmentblock/response/oa_response.html]: Does this not need to be translated?

**Comment by dianakhuang** [comment]
Other than that, :+1: from me.

**Comment by cahrens** [comment]
@andy-armstrong when I press "Continue Assessing Peers" in Turbo mode, nothing happens.

https://andy-armstrong.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/a4dfec19cf9b4a6fb5b18be6ccd9cecc/4dbd99f691064633a9be0daadb805ce6/

File: openassessment/xblock/static/js/src/lms/oa_peer.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ OpenAssessment.PeerView.prototype = {
 
     **/
     continueAssessmentEnabled: function(enabled) {
-        var button = $('#peer-assessment__continue__grading', this.element);
+        var button = $('.action--continue--grading', this.element);
```
Comment (cahrens) [inline openassessment/xblock/static/js/src/lms/oa_peer.js]: Add a Jasmine test for this button. Thanks, @dianakhuang!

**Comment by cahrens** [comment]
:+1: after you address the above feedback and run the bok choy tests (and manually make sure that "Turbo mode" is indeed working).

**Comment by andy-armstrong** [comment]
I've updated the sandbox with the regenerated JS (thanks @dianakhuang!) and the button now works:

https://andy-armstrong.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/a4dfec19cf9b4a6fb5b18be6ccd9cecc/4dbd99f691064633a9be0daadb805ce6/

I'm working on a unit test.

**Comment by dianakhuang** [comment]
:+1: from, me on the updates.

**Comment by andy-armstrong** [comment]
@cahrens I've pushed up changes for the feedback and am running the acceptance and a11y tests now. I'm going to squash and then merge as you gave me thumbs, but feel free to re-review if you'd prefer.

**Comment by cahrens** [comment]
:+1:

**Pull request merged**